### DISCUSSION TITLE
Resize to the half of the center.

### DISCUSSION THREAD

**Discussion by webvars** [rxhanson/Rectangle#539] [Ideas]
How to resize to the half of the width and height, and also place to the center?

**Comment by rxhanson** [comment]
There are a number of ways to do this:

1. You can adjust the "Almost Maximize" size to give you this via terminal commands. https://github.com/rxhanson/Rectangle#almost-maximize
2. You can add an extra centering command with a custom size via terminal commands. https://github.com/rxhanson/Rectangle#add-an-extra-centering-command-with-custom-size
3. You can use my paid superset of Rectangle, Hookshot (https://hookshot.app) to create a custom shortcut.
